.github/workflows/refresh-custom-action-image.yml

```diff
@@ -0,0 +1,52 @@
+# Refresh the docker image that builds specs.  For example, pull in fresh Bikeshed data files.
+name: refresh-custom-action-image
+
+on:
+  schedule:
+    - cron: '0 0 * * 5'
```

Comment:
We may need to trigger this manually sometimes, like when Bikeshed's database changes. We can't control the bikeshed/data versions specref uses, and sometimes in order to fix the specref build we need to do something which breaks a pinned build. So, all that to say, we need to document how to manually trigger this action, perhaps both as a comment in this file and in tools/custom-action. Maybe there's just a button for it on GitHub?